.github/workflows/windows.yml

```diff
@@ -0,0 +1,36 @@
+name: Windows
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: ~\AppData\Local\pip\Cache
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements/**.txt') }}
+          restore-keys: ${{ runner.os }}-pip-
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip setuptools wheel
+          python -m pip install --requirement requirements/pkgutils.txt
+
+      - name: Test
+        run: tox -vv -e ${{ matrix.python-version }}-unit
+        
+      - name: Build
+        run: python setup.py bdist_wheel
+
+      - uses: actions/upload-artifact@v2
```

Comment:
@auvipy do we need this step? celery ships universal wheels so it seems redundant

Reply:
seems redundant